packages/blocks/src/lengthHelpers.ts

```diff
@@ -101,27 +103,40 @@ export const truncators: (
 
 /**
  * mostly internal function for building blocks - takes the element `obj` and
- * applies the truncation `functions` to each limited field iff that field's length is greater than its
- * respective limit for that block
- * @param obj
- * @param functions
- * @param limits
+ * applies the truncation `functions` to each limited field iff that field's
```

Comment:
iff -> if

Reply:
Yeah, this is a math notation thing (if and only if), but as I wrote it I thought that it might be lost on people/look like a typo. I'll just update it.